internal/pkg/config/x509_test.go

```diff
@@ -15,30 +16,52 @@
 package config
 
 import (
+	"path/filepath"
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
+var testJSONFileName = "./res/pkisetup-test.json"
```

Comment:
can we move this to testdata instead of res?

Reply:
ok